### ISSUE TITLE
Narrated comic books in epub

### ISSUE THREAD

**Issue by m-abs** [w3c/publishingcg#20]
Hi,

I'm working as a developer for [Nota](https://nota.dk/om-nota/english) (the Danish Library and Expertise Center for people with print disabilities). We provide accessibility reading material for Danes, who cannot read ordinary printed text. 

We''re in progress of moving from DAISY 2.02 to EPUB 3.

We have a number of narrated comic books in our collection, in our own custom DAISY 2.02 format (I can provide some examples if needed), but we want to use a standard format to distribute them in the future.

I've been looking at [EPUB Region Based Narration](http://idpf.org/epub/renditions/region-nav/) and the [EPUB 3.3 draft](https://www.w3.org/TR/epub-33/) which includes [comics](https://www.w3.org/TR/epub-33/#comics). 

But I don't see how to combine it with [media-overlays](https://www.w3.org/TR/epub-33/#sec-media-overlays).

Is `Region Based Narration` + `Media Overlays` the way to go?
If so how are they combined?
If not, is there a better way to do it?
What about reading system support?

**Comment by dalerrogers** [comment]
I work in multimedia. Watching.

Best Regards,

Dale Rogers, M.Ed., CIW
Assistant Professor, Web and Multimedia Design
Nashville State Community College
Instructional/eLearning Developer
***@***.***
***@***.***
http://dalerogers.me/
https://www.linkedin.com/in/dalerrogers/

From my iPhone. Pardon my thumbs.

________________________________
From: Morten Sjøgren ***@***.***>
Sent: Monday, July 12, 2021 9:13:37 AM
To: w3c/publishingcg ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [w3c/publishingcg] Narrated comic books in epub (#20)


Hi,

I'm working as a developer for Nota<https://nota.dk/om-nota/english> (the Danish Library and Expertise Center for people with print disabilities). We provide accessibility reading material for Danes, who cannot read ordinary printed text.

We''re in progress of moving from DAISY 2.02 to EPUB 3.

We have a number of narrated comic books in our collection, in our own custom DAISY 2.02 format (I can provide some examples if needed), but we want to use a standard format to distribute them in the future.

I've been looking at EPUB Region Based Narration<http://idpf.org/epub/renditions/region-nav/> and the EPUB 3.3 draft<https://www.w3.org/TR/epub-33/> which includes comics<https://www.w3.org/TR/epub-33/#comics>.

But I don't see how to combine it with media-overlays<https://www.w3.org/TR/epub-33/#sec-media-overlays>.

Is Region Based Narration + Media Overlays the way to go?
If so how are they combined?
If not, is there a better way to do it?
What about reading system support?

**Comment by m-abs** [comment]
Could it be as simple as using `<image>` in the [SyncMedia 1.0](https://w3c.github.io/sync-media-pub/sync-media.html#image)? 

Something like this:
```xml
<smil xmlns="http://www.w3.org/ns/SMIL">
    <body>
        <par>
            <audio src="chapter01.mp3" clipBegin="30" clipEnd="40"/>
            <image src="chapter01.jpeg#xywh=percent:5,5,15,15"/>
        </par>
        <par>
            <audio src="chapter01.mp3" clipBegin="40" clipEnd="50"/>
            <image src="chapter01.jpeg#xywh=percent:20,20,25,25"/>
        </par>
        <par>
            <audio src="chapter01.mp3" clipBegin="50" clipEnd="60"/>
            <image src="chapter01.jpeg#xywh=percent:5,45,30,30"/>
        </par>
    </body>
</smil>
```

This probably doesn't solve it for more complex comic books, but it could serve as a temporary solution for our needs.

**Comment by Jeffxz** [comment]
Hi @m-abs, do you have some "real" example of a few images, region defined by "chapter01.jpeg#xywh=percent:20,20,25,25" or "left: 145px; top: 127px; width: 1813px; height: 749px;" and audio clip? I probably can help experiment something? I think this could also relate to potential annotation for comic.

**Comment by Jeffxz** [comment]
I also created a discussion related to this https://github.com/w3c/publishingcg/discussions/29

**Comment by m-abs** [comment]
> Hi @m-abs, do you have some "real" example of a few images, region defined by "chapter01.jpeg#xywh=percent:20,20,25,25" or "left: 145px; top: 127px; width: 1813px; height: 749px;" and audio clip? I probably can help experiment something? I think this could also relate to potential annotation for comic.

Hi @Jeffxz 

"chapter01.jpeg#xywh=percent:20,20,25,25" was just a suggestion, we don't have a real life example of it.

"left: 145px; top: 127px; width: 1813px; height: 749px;" is how we make the comics in our `DAISY 2.0.2` books.
You can download one of our free comicbooks[here](https://nota.dk/bibliotek/bog/gravr%C3%B8verne#audio) select "Daisy" in the dropdown to the left of "Download"

**Comment by gautierchomel** [comment]
I would still prefer to migrate (even if to be closed directly) so that all issues are grouped in one place.
